submissions/peterIyiola/database/init.sql

```diff
@@ -0,0 +1,132 @@
+('LG'),
+('Hisense'),
+('Midea'),
+('panasonic');
+
+create table products (
+	id int not null auto_increment,
+	name varchar(50) not null,
+	description varchar(250) not null,
+	image blob not null,
+    category_id int not null,
+    admin_id int not null,
+	unit_price decimal(5,0) not null,
+    stock_level int not null,
+    status varchar(20) not null,
+	primary key (id),
+    foreign key (category_id) references categories (id) on delete cascade,
+    foreign key (admin_id) references best_buyers_admin (id) on delete cascade
+);
+/--- insert values into products --/
+insert into products (name, description, image, category_id, admin_id, unit_price, stock_level, status)
+values('samsung 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg',500000.00, 2, 4,180, 'in stock'),
+('LG 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg',400000.00, 5, 4, 120, 'in stock'),
+('Hisense 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg',300000.00,6,4,130, 'in stock'),
+('Panasonic 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg',200000.00, 4, 4, 150, 'in stock'),
+('Thermocool 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg',300000.00,3,4,160, 'in stock');
+
+/--- created customers table --/
+create table customers (
+	id int not null auto_increment,
+	first_name varchar(40) NOT NULL,
+	last_name varchar(20) NOT NULL,
+	email_address varchar(50) NOT NULL,
+	password varchar(30) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+/--- insert values into customers  --/
+insert into customers (first_name, last_name ,email_address, password)
+values('Patrick', 'Isreal','patricki@gmail.com','p745377'),
+('tommy' ,'hagen','tm@gmail.com','TM52637'),
+('Jamie' ,'vardy','jv@gmail.com','jvte7e878832'),
+('richard' ,'Hanger','rm@gmail.com','TM52yry73'),
+('Samuel' ,'etuhu','se@gmail.com','TM5=7445');
+
+/--- create table for customer details --/
+create table customer_addresses (
+	id int not null auto_increment,
+	customer_id int null,
+	street_name varchar(100) not null,
+	city varchar(20) not null,
+	state varchar(20) not null,
+	zip_code int,
+	country varchar(20) not null,
+	phone_number varchar(15) not null,
+    primary key (id),
+	foreign key (customer_id) references customers (id) on delete cascade
+);
+
+insert into customer_addresses (customer_id, street_name,city,state,zip_code,country,phone_number)
+values('1','abejoye','VI','lagos',100449,'nigeria',08074949934),
+('2','akinlabake','ishaga','lagos',100949,'nigeria',08074945930),
```

Comment:
id's are `int` and not `varchar` hence is shouldn't be wrapped around `''`. Eg `1`